samples/benchmarks/coremark/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_COREMARK_ITERATIONS=4000
```

Comment:
add license header

Reply:
Also consider adding the same for `nrf54l15pdk_nrf54l15_cpuapp_ns`